steal.lic

```diff
@@ -134,6 +137,19 @@ class Steal
     move('go door')
   end
 
+  def walk_to_bin
+    case @hometown
+    when 'Crossing'
```

Comment:
This feels like something that should be in `base-town.yaml`. What do you think?